test/functional/basic/basic-e2e-specs.js

```diff
@@ -202,8 +203,12 @@ describe('XCUITestDriver - basics', function () {
     });
 
     it('should start a session, navigate to url, get title', async () => {
-      let contexts = await driver.contexts();
-      contexts.length.should.be.at.least(2);
+      let contexts;
+      await retryInterval(10, 1000, async () => {
+        // on some systems (like Travis) it takes a while to load the webview
+        contexts = await driver.contexts();
+        contexts.length.should.be.at.least(2);
```

Comment:
If this interval function needs to be called `n` times, won't the first `n-1` calls fail and break the test?